lib/CFRPageActions.jsm

```diff
@@ -57,6 +57,9 @@ class PageAction {
     this.stateTransitionTimeoutIDs = [];
 
     this.container.onclick = this._handleClick;
+    // Collapse the recommendation on url bar focus in order to free up more
+    // space to display and edit the url
+    this.urlbar.addEventListener("focus", this._collapse);
```

Comment:
Hmm......... I see that `PageActionMap` is a `WeakMap` but this event listener might create a reference cycle leading to a leak. Maybe the original code location inside `show` would work with a matching clean up in `hide`? (I do also see that onclick has callback reference already, so we might already be leaking… @k88hudson ?)

Also, `_collapse` will be called with the `event` and it's checking for truthiness right now, so maybe just change the check for number-ness to delay?